beets/ui/__init__.py

```diff
@@ -773,6 +774,26 @@ def add_all_common_options(self):
 # succinct example program.
 
 
+# FIXME: kill original Subcommand in favor of this
```

Comment:
I feel like I need some help with this: Is backwards-compatibility for the plugin API important?